File: c-sharp/gameoflife/Cell.cs
namespace gameoflife
{
    public class Cell
    {
        public Cell()
        {
            IsAlive = false;
        }
        public Cell(bool isalive)
        {
            IsAlive = isalive;
        }
        
        public bool IsAlive { get; set; }
        public bool Lives(int neighbors)
        {
            if (neighbors == 2 || neighbors == 3) {
                return true;
            }
            return false;
        }
    }
}

File: java/src/main/java/Cell.java
public class Cell {

    public boolean isAlive() {
        return isAlive;
    }

    public void setAlive(boolean alive) {
        isAlive = alive;
    }

    public boolean Lives(int neighbors) {
        if (neighbors == 3) {
            return true;
        }
        if (neighbors == 2) {
            return isAlive;
        }
        return false;
    }

    private boolean isAlive;

    public Cell(boolean alive) {
        isAlive = alive;
    }

}
